web/src/classic/components/organisms/Published/core/hooks.ts

```diff
@@ -54,6 +58,22 @@ export default (alias?: string) => {
 
   const tags = data?.tags; // Currently no need to convert tags
 
+  const selectedLayerId = useMemo(() => {
+    return selected?.type === "layer"
+      ? { layerId: selected.layerId, featureId: selected.featureId }
+      : undefined;
+  }, [selected]);
+
+  const selectLayer = useCallback(
```

Comment:
handleLayerSelect

Reply:
As we discussed we will keep it as is for classic.